$MirOS: ports/games/bsdgames/patches/patch-rogue_rogue_6,v 1.1.7.1 2005/03/18 15:46:07 tg Exp $
--- rogue/rogue.6.orig	Thu Jan  1 00:00:00 1970
+++ rogue/rogue.6	Thu Feb 22 21:51:08 2007
@@ -50,7 +50,7 @@ fantasy games.
 .Pp
 To get started you really only need to know two commands.
 The command
-.Ic ?\&
+.Ic \&?
 will give you a list of the available commands and the command
 .Ic \&/
 will identify the things you see on the screen.
